images/openstack/Dockerfile.ci

```diff
@@ -29,7 +29,15 @@ RUN yum update -y && \
     python3-openstackclient ansible-2.9.14-1.el8ae python3-openstacksdk python3-netaddr unzip jq nmap && \
     yum clean all && rm -rf /var/cache/yum/*
 
-RUN python -m pip install yq
+# ansible 2.9 is EOL in September 2023, so we need to install ansible-core and get the collections from source
+# until we have a package available.
+RUN ansible-galaxy collection install openstack.cloud ansible.utils community.general && \
+    mkdir -p /usr/share/ansible/collections/ansible_collections && \
+    cp -r /root/.ansible/collections/ansible_collections/* /usr/share/ansible/collections/ansible_collections/
+
+RUN python3 -m pip install --upgrade pip
+# ansible-core comes with python3.8 but openstacksdk comes with python3.6 so let's install them from pip.
+RUN python3 -m pip install yq openstackclient openstacksdk netaddr
```

Comment:
Meh, [that step fails](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_installer/7385/pull-ci-openshift-installer-release-4.13-e2e-openstack-upi/1686184553086980096#1:build-log.txt%3A1499):
```
ERROR: Cannot uninstall 'PyYAML'. It is a distutils installed project and thus we cannot accurately determine which files belong to it which would lead to only a partial uninstall.
error: build error: building at STEP "RUN python3 -m pip install yq openstackclient openstacksdk netaddr": while running runtime: exit status 1 
```

Reply:
I found the issue. My backport wasn't clean. Now it should be better. Let's see how CI works this time.